resources/.ubsan-blacklist.txt

```diff
@@ -6,4 +6,7 @@
 [undefined]
 # The hash map seems to reinterpret memory addresses in some semi-legal (but working) way. Example:
 # ska::bytell_hash_map<int, int> h; h.emplace(11, 11); std::cout << h[11] << std::endl;
-src:*bytell_hash_map.hpp
\ No newline at end of file
+src:*bytell_hash_map.hpp
+
+# A false positive in LZ4
```

Comment:
Are we sure that this is a false positive or could this also be an error in LZ4?

Reply:
Might also be an actual issue in LZ4, but nothing that we can fix. Added a link to https://github.com/lz4/lz4/issues/847

Reply:
Having seen a minimal example outside of LZ4, this is simply a new check that the LZ4 cannot work with. Quite sure that this is not a bug.